### ISSUE TITLE
Multi-select tabs in sidebar

### ISSUE THREAD

**Issue by Lotus49b** [oxdc/obsidian-vertical-tabs#32]
This would be helpful in quickly selecting a number of tabs at once for deletion or rearranging.

Thanks for a very useful plugin.

**Comment by oxdc** [comment]
Thank you. Multi-selection will come after a structural refactoring. Please stay tuned 🎉

**Comment by greetclammy** [comment]
Looking forward to this!

**Comment by nemonein** [comment]
Does it work on the current version?
I'm on 0.17.7 but cannot select multiple tabs.
It would be very nice if I could select files with `SHIFT` or `CTRL` + click.

**Comment by churnish** [comment]
@nemonein I can select multiple tabs in the plugin pane with Cmd on macOS.

**Comment by nemonein** [comment]
@churnish Thank you. I'm on linux, and `Ctrl` with click works. `Shift` does not.
I think I only tried with `Shift` keys.

**Comment by nemonein** [comment]
Ah... I can select files, but ALL that I can do is move the files to another group. I cannot close/delete the files.
Is it possible?

**Comment by oxdc** [comment]
> Ah... I can select files, but ALL that I can do is move the files to another group. I cannot close/delete the files. Is it possible?

Hi @nemonein ! I am still working on this feature.

**Comment by nemonein** [comment]
@oxdc Thanks. I'm looking forward to it.
And I'd love for the `Shift` key to work as well.

**Comment by churnish** [comment]
I agree with @nemonein. I'd like the Shift key to work like in the does in the file explorer. When a file is selected and I press on another file while holding down Shift, all files in-between get selected:

<img width="1200" height="775" alt="Image" src="https://github.com/user-attachments/assets/8fad9a2c-89a8-48a3-a201-bc715d762ac7" />

**Comment by oxdc** [comment]
Hi @nemonein @churnish ! Shift-key selection and the multi-select menu are now available for [Beta Program](https://vertical-tabs-docs.oxdc.dev/Beta-Versions/beta-program) subscribers. Will be available to everyone later.
